versions/build.jet.ver

```diff
@@ -1,2 +1,3 @@
 export stack_intel_ver=2021.5.0
 export stack_impi_ver=2021.5.1
+source "${HOMEgfs:-}/versions/build.spack.ver"
```

Comment:
If you want to source `build.spack.ver` and not combine them at the link step, then `build.spack.ver` should be sourced first -- it provides the defaults which can be overridden by each machine file.  That's the way I coded it, at least.

Reply:
I think this should be reworked so machine-specific versions are not set in `build.spack.ver`.  I've opened issue #2123 to do so.  This doesn't hurt this PR, though, so I will resolve this conversation.